OptimizelySDK/Optimizely.cs

```diff
@@ -62,6 +68,7 @@ public class Optimizely
             ErrorHandler = errorHandler ?? new NoOpErrorHandler();
             Bucketer = new Bucketer(Logger);
```

Comment:
EventBuilder requires Bucketer and we are passing the reference of Bucketer to Decision Service. Should I change the logic? Remove Bucketer from Optimizely class, create Bucketer object in Decision Service & then where EventBuilder object is being used, I must set reference of Bucketer object from the DecisionService object to EventBuilder object?